res/values/strings.xml

```diff
@@ -13748,4 +13756,19 @@
 
     <string name="battery_share_description">Charge other devices by placing them on the back of your phone</string>
     <string name="battery_share">Battery share</string>
+
+    <string name="choose_duress_password_title">Choose a new Duress Lock</string>
+    <string name="update_duress_password_title">Update existing Duress Lock</string>
+
+    <string name="duress_pin_exist">PIN</string>
+    <string name="duress_password_exist">PASSWORD</string>
+    <string name="duress_pin_and_password_exist">PIN &amp; PASSWORD</string>
+    <string name="added_a_duress_pin_and_password">NONE</string>
+
+    <string name="duress_descriptions">Duress mode in GrapheneOS will only work on if you are using a lockscreen which is the same as what you have configured for duress. Duress mode will wipe the device fully, including user data and eSIMS. Duress mode does not erase the presence of GrapheneOS of the device. After duress mode has been triggered by the user, the device will act as if it is a fresh installation of GrapheneOS.</string>
+    <string name="duress_title">Add duress password?</string>
+    <string name="delete_duress_title">Delete duress password?</string>
+    <string name="delete_duress_descriptions">Duress password in GrapheneOS allow you to quickly wipe your device without unlocking the device.</string>
```

Comment:
Fix: `allow` should be `allows`
Consider removing `in GrapheneOS` as it seems unnecessary
Change `wipe your device without unlocking the device.` to `wipe your device without unlocking it.`